src/components/FilterDrink.js

```diff
@@ -49,4 +49,4 @@ const FilterDrink = ({drinksData, onSortDrink}) => {
 
```

Comment:
**4 line of code** - You don't use `sort`.
**27 line of code** - You don't need `updatedContent`.
**38 line of code** - `;`.
**44 - line of code**:

```
   <li
        key={tab.id}
        className={`${tab.status ? 'active' : ''} filter-item`}
        onClick={() => onFilterDrink(tab.category)}
    >
        {tab.name}
  </li>
```

**49 line of code** - empty.
**50 line of code** - `;`.